.tekton/main-pull-request.yaml

```
@@ -299,6 +299,22 @@ spec:
           image: registry.access.redhat.com/ubi8/ubi:latest
           script: exec "$(workspaces.source.path)/source/scripts/konflux/subscription-manager/subscription-manager-bro.sh" smuggle
 
+    - name: fetch-external-networks
+      runAfter:
+      - clone-repository
+      taskSpec:
+        steps:
+          - name: fetch-external-networks
+            image: registry.access.redhat.com/ubi8-minimal:latest
+            script: |
+              #!/usr/bin/env bash
+              microdnf install -y openssl zip
+              "$(workspaces.source.path)/source/image/rhel/fetch-stackrox-data.sh" \
+                  "$(workspaces.source.path)/source/.konflux/stackrox-data"
```

[minor and optional] Since it's the last command of the script, you can `exec` into it which saves from creating a child process.
```suggestion
              exec "$(workspaces.source.path)/source/image/rhel/fetch-stackrox-data.sh" \
                  "$(workspaces.source.path)/source/.konflux/stackrox-data"
```